addons/pkg/util/addon_util.go

```diff
@@ -70,6 +73,11 @@ func GenerateAppSecretNameFromAddonSecret(addonSecret *corev1.Secret) string {
 	return fmt.Sprintf("%s-data-values", GenerateAppNameFromAddonSecret(addonSecret))
 }
 
+// GenerateDataValueSecretNameFromAddonAndClusterNames generates data value secret name from addon names
```

Comment:
nit: I think you were intending to use the name in comment for function name.Semantically it sounds better to me.
You can choose to update comment or function name.